lib/pages/Request_Ride_Loc.dart

```diff
@@ -119,7 +80,7 @@ class _RequestRideLocState extends State<RequestRideLoc> {
                 children: <Widget>[
                   Flexible(
                     child: Text("Where do you want to go?",
-                        style: RideRequestStyles.question(context)),
+                        style: TextThemes.question),
```

Comment:
The way rider and driver have structured their style guides seem very different. Before either theme is merged in, they should get aligned for consistency (driver names their themes based on the names in the figma design guide, this one does not)